packages/at_secondary_server/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 3.0.49
+- feat: Enforce superset access check for approving apps
+- fix: respect isEncrypted:false if supplied in the notify: command, and 
+  ensure that the correct value is always transmitted onwards
+
```

Comment:
No mention of the beta status removal below.

Reply:
> No mention of the beta status removal below.

added